portfolio/src/main/webapp/index.html

```diff
@@ -20,11 +20,11 @@ <h1 id="title"> My Portfolio</h1>
             </div>
             <div id="tabSelector">
                 <div id="tabs">
-                    <div role="button" class="tab tabSelected" id="Welcome"><p class="tabName">Welcome</p></div>
-                    <div role="button" class="tab" id="General"><p class="tabName">General</p></div>
-                    <div role="button" class="tab" id="Education"><p class="tabName">Education</p></div>
-                    <div role="button" class="tab" id="Projects"><p class="tabName">Projects</p></div>
-                    <div role="button" class="tab" id="Gallery"><p class="tabName">Gallery</p> </div>
+                    <div role="button" title="welcomeTab" class="tab tabSelected" id="Welcome"><p class="tabName">Welcome</p></div>
```

Comment:
The title attribute is shown in a tooltip when you hover your mouse over the element. It should be human readable, so I wouldn't use camelCase here.